### ISSUE TITLE
Relative paths in prompt configuration isn't resolved

### ISSUE THREAD

**Issue by chrisprice** [promptfoo/promptfoo#1063]
In previous versions (tested with 0.61.0), the following configurations were accepted -

```
prompts:
  - file://../../some/path/to/a/prompt.txt
```

```
prompts:
  - prompts/prompt.txt
```

However, this is not resolved in the latest version -

```
    return binding.readFileUtf8(path, stringToFlags(options.flag));
                   ^

Error: ENOENT: no such file or directory, open 'prompt.txt'
    at Object.readFileSync (node:fs:453:20)
    at processTxtFile (/opt/homebrew/lib/node_modules/promptfoo/dist/src/prompts/processors/text.js:38:28)
    at processPrompt (/opt/homebrew/lib/node_modules/promptfoo/dist/src/prompts/index.js:117:42)
    at readPrompts (/opt/homebrew/lib/node_modules/promptfoo/dist/src/prompts/index.js:136:35)
    at resolveConfigs (/opt/homebrew/lib/node_modules/promptfoo/dist/src/main.js:129:59)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async runEvaluation (/opt/homebrew/lib/node_modules/promptfoo/dist/src/main.js:545:48)
    at async Command.<anonymous> (/opt/homebrew/lib/node_modules/promptfoo/dist/src/main.js:704:9) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'prompt.txt'
}
```

The [documentation](https://www.promptfoo.dev/docs/configuration/parameters/#prompts-from-file) suggests this should work.
I believe the source of the change was [this commit](https://github.com/promptfoo/promptfoo/commit/9bd03d29e3179f2b665c867d3cd4528b2c056264#diff-0f0b37407b21be6434ac29c4b75a5a1c31ade0668c1afafa267cba4efc13aba9).

**Comment by mldangelo** [comment]
Thanks for reporting this issue, @chrisprice. I apologize for the inconvenience caused by this bug (introduced in [PR #994](https://github.com/promptfoo/promptfoo/pull/994)). I've addressed the problem with relative paths in [PR #1066](https://github.com/promptfoo/promptfoo/pull/1066). I'll get a fix deployed later today. Please let me know if it doesn't resolve the issue for you. Thanks for your patience and thanks again for reporting this!

**Comment by mldangelo** [comment]
A fix for this was deployed in [0.68.1](https://github.com/promptfoo/promptfoo/releases/tag/0.68.1) Please let me know if you run into any other issues and thanks again for reporting it!
